src/Features/Core/Portable/QuickInfo/QuickInfoSectionKinds.cs

```diff
@@ -16,5 +16,6 @@ public static class QuickInfoSectionKinds
         public const string Exception = nameof(Exception);
         public const string Text = nameof(Text);
         public const string Captures = nameof(Captures);
+        internal const string NullabilityAnalysis = nameof(NullabilityAnalysis);
```

Comment:
Making this internal as this is still considered a preview experience and indeed may change. Will make it public in the 16.3 branch if this PR ships in 16.2. If it first goes into 16.3 then I'll change this prior to merge.

Reply:
Make sure to add a follow up issue